definitions/divorce/json/CaseField/CaseField-general-referral-nonprod.json

```diff
@@ -84,5 +84,32 @@
     "Label": "General referral decision date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GeneralReferralPaymentDetails",
+    "Label": "How will payment be made?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "PaymentTypeEnum",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferralPaymentFeeSummary",
```

Comment:
If possible this should start with a capital letter to follow convention for field names.

Reply:
Updating now